### DISCUSSION TITLE
Ignoring examples when building docs in pipeline

### DISCUSSION THREAD

**Discussion by AVHopp** [emdgroup/baybe#69] [Documentation]
Hey @AdrianSosic @Scienfitz 
I tried a bit if there is a clever and simple way to make sphinx ignore all links related to examples but did not find an easy and quick-to-implement way. Thus my question how to deal with this:

1. Agree on not putting docs in the CI pipeline in general
2. Just build the examples in the pipeline
3. Agree on not putting it there for now and seeing whether we want to include this at a later point
4. Try a more complex solution that would require more modifications

Which ones are your favorites?

**Comment by AdrianSosic** [comment]
Hey @AVHopp, before we go into your 4 suggested alternatives, I think I have an idea that could make it work in a very easy and stupid way: how about you just delete the content of `examples` before running sphinx!? I think this should work out of the box, because that's equivalent to "yes, build the examples, but there are actually none". So no need to adjust any settings...

**Reply by AVHopp** [reply]
That is actually a good idea. For sphinx, it would be fine if we just write the markdown files without executing the examples. I'll give this a try, thanks for the hint :)
This would basically just change the way that "ignore_examples" works - it would then create the files, but not execute the code, and should hopefully work even without the force option.

**Comment by Scienfitz** [comment]
closing as completed
